lldb/packages/Python/lldbsuite/test/lang/swift/step_into_override/TestStepIntoOverride.py

```diff
@@ -1,16 +1,74 @@
-# TestStepIntoOverride.py
-#
-# This source file is part of the Swift.org open source project
-#
-# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
-# Licensed under Apache License v2.0 with Runtime Library Exception
-#
-# See https://swift.org/LICENSE.txt for license information
-# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-#
-# ------------------------------------------------------------------------------
-
-import lldbsuite.test.lldbinline as lldbinline
+import lldb
 from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import platform
+import unittest2
 
-lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest])
+class TestStepIntoOverride(lldbtest.TestBase):
+    mydir = lldbtest.TestBase.compute_mydir(__file__)
+
+    @swiftTest
+    @skipIfLinux
```

Comment:
I'm surprised this is necessary.

Reply:
Yes, you're right. The `skipIfLinux` should go away. Thanks!